source/frontend/src/features/contacts/contact/edit/Organization/UpdateOrganizationForm.tsx

```diff
@@ -59,28 +52,11 @@ export const UpdateOrganizationForm: React.FC<{ id: number }> = ({ id }) => {
   // fetch organization details from API for the supplied Id
   const { organization } = useOrganizationDetail(id);
   const formOrganization = useMemo(
-    () => apiOrganizationToFormOrganization(organization),
+    () => IEditableOrganizationForm.apiOrganizationToFormOrganization(organization),
```

Comment:
An interface with static methods smells fishy.

Reply:
Thats true, is a class now. Thought about renaiming it to match the current pattern we have but that could show more diffs. I plan on making a new pr with the renames given this one has a lot of changes already